app/components/SideBarMenu.tsx

```diff
@@ -28,13 +28,13 @@ function navMenuItems(): MenuItem[] {
 				title: "nBTC (testnet)",
 				link: "",
 				subNavItems: [
-					{
-						id: "navigation-2-1",
-						icon: "",
-						subNavItems: [],
-						link: "/",
-						title: "Buy or Sell nBTC",
-					},
+					// {
+					// 	id: "navigation-2-1",
+					// 	icon: "",
+					// 	subNavItems: [],
+					// 	link: "/",
+					// 	title: "Buy or Sell nBTC",
+					// },
```

Comment:
use /* */ comments - it's more erognomic, and explain why we are commenting out.